src/main/kotlin/value/VObj.kt

```diff
@@ -11,7 +12,7 @@ data class VObj(val v: ULID?): Value() {
     @SerialName("yType")
     override val type = Type.OBJ
 
-    override fun toString() = "#$v"
+    override fun toString() = "#${v.toString().takeLast(5)}"
```

Comment:
This is just a readability convenience for me for debugging, so when I test things I can quickly check by eye if some obj value is what I expected.

Reply:
Just curious, do you plan to have a short-hand way to call out objects in code, like we did in MOO (`#xxx`)?  If so, you might want to go even more extreme than ULIDs for the sake of compactness:  https://github.com/viascom/nanoid-kotlin.  I _think_ that would also allow you to have vanity obj IDs, but that would introduce some other complications re: checking uniqueness.

Reply:
Thanks for this suggestion, somehow I forgot about nanoID.  I swapped ULID out for it everywhere after taking a look.  I'm not supporting obj literals yet, but now I totally could, thanks!